### ISSUE TITLE
[A11Y][Alert message] Warning/Error status are read first and with the close button

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#731]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

Android and iPhone

### OS Version

.

### App version

1.3.0 (681)

### Describe the issue

When a status Negative or Warning is set, the status description has an impact on screen reader order and focus: the whole container is first read with the status (Negative or Warning) combined with the close button then focus is moved to label.
<img width="328" height="640" alt="" src="https://github.com/user-attachments/assets/b6d760c5-ef00-425a-bbca-8be824c351d0" />

<img width="315" height="637" alt="" src="https://github.com/user-attachments/assets/f4685f9d-08a4-4542-add7-8ac9fcb3c6dc" />

### Expected Behavior

Status (Warning or Negative or others if exist) should be read with the Label + desc + bullet list, then after that the close button should be read (same behavior than Neutral status)
